src/extensions/ignite/copyBatch.js

```diff
@@ -38,8 +47,11 @@ module.exports = (plugin, command, context) => {
       // generate the React component
       if (await shouldGenerate(job.target)) {
         const currentPluginPath = ignitePluginPath()
+
+        const sporkDirectory = findSpork(context, job.template)
+
         await template.generate({
-          directory: directory || (currentPluginPath && `${currentPluginPath}/templates`),
+          directory: directory || sporkDirectory || (currentPluginPath && `${currentPluginPath}/templates`),
```

Comment:
I assume someone passing in `directory` should take precedence over a spork. This is an unusual circumstance, I think.